physical/cassandra/cassandra_test.go

```diff
@@ -17,6 +17,10 @@ import (
 )
 
 func TestCassandraBackend(t *testing.T) {
+	if os.Getenv("VAULT_CI_GO_TEST_RACE") != "" {
```

Comment:
I'm not sure what values we expect for `VAULT_CI_GO_TEST_RACE` but generally, I prefer to check for a specific nonempty value, rather than just anything. E.g. if it was `1` someone might reasonably think setting it to `0` would disable it. Likewise if it was `true/false` `YES/NO` etc.

I know that value is a recent change, maybe we should wrap up the checking logic into a single function in a test helper so we could just call that rather than having to write code like this each time?

Reply:
Sure, will do